cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/ProgramLifecycleHttpHandlerTest.java

```diff
@@ -188,6 +188,21 @@ public void testProgramStartStopStatus() throws Exception {
 
     waitState(dummyMR2, STOPPED);
 
+    // start multiple runs of the map-reduce program
+    startProgram(dummyMR2);
+    startProgram(dummyMR2);
+    verifyProgramRuns(dummyMR2, "running", 1);
+    historyRuns = getProgramRuns(dummyMR2, "running");
+    Assert.assertEquals(2, historyRuns.size());
+
+    // stop all runs of the map-reduce program
+    stopProgram(dummyMR2, 200);
+    waitState(dummyMR2, STOPPED);
```

Comment:
The status of the program returned is either STOPPED or RUNNING. I have added a check on previous dummyMR2 test about stopping individual runs to check if the second dummyMR2 is still running after we stop the first run. The result shows it is running so I think we are testing the functionality of stopping all programs once. Can you please check my change again? Thank you!